tests/test_iostream.py

```diff
@@ -8,6 +8,16 @@
 
 from pybind11_tests import iostream as m
 
+if sys.platform == "win32":
```

Comment:
```suggestion
if sys.platform.startswith("win32"):
```

It's canonical to use startswith, though it's often not required.

Reply:
Looking at this with a fresh eye: For a moment I forgot that we have `env.WIN`. Changed to use that: commit 8b79a85b424ff721815e4937cee29017d046b4da